src/locales/fr-FR/translation.json

```diff
@@ -24,6 +24,10 @@
       "body": "Nous n'avons trouvé aucun Cozy à cette adresse. Il s'agit peut-être d'une erreur de frappe. Pour le savoir, vérifiez l'adresse de votre Cozy dans l'e-mail que nous vous avons envoyé au moment de sa création.",
       "title": "L'adresse de ce Cozy n'existe pas."
     },
+    "cozyNotOnboarded": {
```

Comment:
as for the other PRs we'll have to be careful and handle these new strings in the i18n

Reply:
I applied latest translation changes